rest-server/core/src/main/scala/com/actionml/core/store/backends/MongoDao.scala

```diff
@@ -31,19 +31,39 @@ import org.mongodb.scala.MongoCollection
 import org.mongodb.scala.bson.{BsonInt32, BsonValue}
 import org.mongodb.scala.bson.collection.immutable.Document
 import org.mongodb.scala.bson.conversions.Bson
+import org.mongodb.scala.model.{Filters, Sorts}
+import java.time.{Instant, OffsetDateTime, ZoneOffset}
+import java.util.concurrent.TimeUnit
+
+import com.actionml.core.store.{DAO, Ordering, Store}
+import com.mongodb.client.model.IndexOptions
+import com.typesafe.scalalogging.LazyLogging
+import org.bson.codecs.configuration.{CodecProvider, CodecRegistries}
+import org.bson.codecs.{Codec, DecoderContext, EncoderContext}
+import org.bson.{BsonReader, BsonWriter}
+import org.mongodb.scala.bson.collection.immutable.Document
+import org.mongodb.scala.model.IndexModel
+import org.mongodb.scala.{MongoClient, MongoCollection, MongoDatabase}
+
+import scala.concurrent.duration._
+import scala.concurrent.{Await, ExecutionContext, Future}
+import scala.reflect.ClassTag
+import scala.reflect.runtime.universe._
```

Comment:
@andreyshikov did you mean to import all these? I don't see a code change that would require them.